src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write your code here
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+
+const dataObj = rows.map(row => {
+  const cells = Array.from(row.children);
+  const extraPositionCell = document.createElement('td');
+
+  extraPositionCell.textContent = cells[1].textContent;
+
+  return {
+    name: cells[0].textContent,
+    position: cells[1].textContent,
+    office: cells[2].textContent,
+    age: cells[3].textContent,
+    position2: extraPositionCell.textContent,
+    salary: cells[4].textContent,
+  };
+});
+
+const tHeadRow = document.querySelector('thead tr');
+const tFootRow = document.querySelector('tfoot tr');
+const newTh = document.createElement('th');
+
+newTh.textContent = 'Position';
+
+const newThForTfoot = newTh.cloneNode(true);
+
+const lastThInHead = tHeadRow.querySelector('th:last-child');
+const lastThInFoot = tFootRow.querySelector('th:last-child');
+
+tHeadRow.insertBefore(newTh, lastThInHead);
+tFootRow.insertBefore(newThForTfoot, lastThInFoot);
+
+const tbody = document.querySelector('tbody');
+
+tbody.innerHTML = '';
```

Comment:
Clearing the innerHTML of tbody is not necessary, you can directly append the new cell to each row.